config.gradle.kts

```diff
@@ -0,0 +1,16 @@
+extra["major"] = 0
+extra["minor"] = 0
+extra["patch"] = 1
+
+extra["buildConfig"] = mapOf(
+    "minSdk" to 28,
+    "compileSdk" to 33,
+    "targetSdk" to 33
+)
+
+extra["releaseConfig"] = mapOf(
+    // TODO - com.infinum.<YOUR-AWESOME-LIBRARY-NAME>
+    "group" to "com.infinum.libname",
+    "version" to "${extra["major"]}.${extra["minor"]}.${extra["patch"]}",
+    "versionCode" to (extra["major"] as Int * 100 * 100 + extra["minor"] as Int * 100 + extra["patch"] as Int)
```

Comment:
1. This version code generation snippet is generally problematic for various reasons.
2. I don't think we need to version the sample app anyway, it can always have the version code of 1 and that's okay.
3. The version of the library will be more readable if inlined like before (0.0.1 as opposed to major = 0, etc.).

Reply:
> I don't think we need to version the sample app anyway, it can always have the version code of 1 and that's okay.

Agreed, but I still thing version code may be necessary for some libraries that are technically an application on their own, ie: Thimble.

Reply:
Those libraries can then implement this themselves. I would say that is an exception, not the rule, and we here soft-force most libraries to conform to that exception rather than defaulting to the simpler, most common implementation.

Reply:
@antunflas also commented on this here: https://github.com/infinum/android-library-template/pull/1#discussion_r946547181